### PR TITLE
feat: add AI category icon

### PR DESCRIPTION
**Pull request by ghost**
## Summary

Add left menu icon for the AI products.
Ask if you need the related jira ticket.

## Type

- Feature

#### What is expected?

(Description of the new behavior)

#### The following changes where made:

(Describe what you did)

1.

2.

3.

### CODE REVIEW

**Review by dmzski — CHANGES_REQUESTED** [review]
The viewBox of the icon is not well configured it should be `0 0 18 18`

File: src/components/Icon/index.tsx [hunk 1]
```diff
@@ -8,6 +8,15 @@ import Box, { XStyledProps } from '../Box'
 // Non Material Design icons: 'send',
 
 const ICONS = {
+  ai: () => (
+    <g fill="none" fillRule="evenodd">
```
Comment (chambo-e) [inline src/components/Icon/index.tsx:12]: Could you pass the svg through https://jakearchibald.github.io/svgomg/ to optimise it pls ? :)

**Review by chambo-e — CHANGES_REQUESTED** [review]

File: src/components/Icon/index.tsx (hunk 1 above)
Comment (David-Chaille) [reply src/components/Icon/index.tsx:12]: Done using the default settings : 32.65%

**Comment by David-Chaille** [comment]
> The viewBox of the icon is not well configured it should be `0 0 18 18`

can you confirmed that it's ok now ?

File: src/components/Icon/index.tsx (hunk 1 above)
Comment (chambo-e) [reply src/components/Icon/index.tsx:12]: I can't seem to have the same result as you

```
<svg width="20" height="20" fill="none" xmlns="http://www.w3.org/2000/svg">
  <path d="M17.7 11.9v-.2l.1-.6a2.8 2.8 0 00-1-2.3 3 3 0 00-.4-3.4c-.5-.5-1.1-.8-1.8-1a4 4 0 00-2.5-2.2c-.7-.3-1.4-.3-2.1 0a2.3 2.3 0 00-1 2.4v10a3.2 3.2 0 002.7 3.4h.4a3 3 0 002.8-1.6 2.5 2.5 0 001.4-1.5c.2-.5.3-1 .2-1.5.5-.3 1-.9 1.2-1.5zm-3 2.3a.7.7 0 01-.6.5 1 1 0 00-.8.7 1.2 1.2 0 01-1.4.8c-.6 0-1-.6-1-1.6V12l.1-.7a.8.8 0 01.8-.5 1 1 0 000-1.8 3 3 0 00-1 .1V4.6v-.8l.7.1a2 2 0 011.2 1c-.5.3-.8.8-1 1.3a1 1 0 001.6.8c.2-.5.4-.7.7-.7a1 1 0 011 .3 1.3 1.3 0 01-.2 1.9 1 1 0 00.4 1.4l.8.8v.4l-.1.4-.2.2a1.1 1.1 0 01-.9.2 1 1 0 00-.5 1.8h.3a.6.6 0 010 .5z" fill="#AE5DF6"/>
  <path d="M10.8 14.6V4.7a2.4 2.4 0 00-1-2.5c-.6-.3-1.4-.3-2 0a4 4 0 00-2.6 2.3c-.6 0-1.3.4-1.7.9A3 3 0 003 8.8c-.6.5-1 1.1-1 1.8a2.5 2.5 0 00.7 2.5l.6.3c-.1.5 0 1 .1 1.5A2.5 2.5 0 005 16.4 3 3 0 007.7 18h.5a3.2 3.2 0 002.6-3.4zM8 16.2c-1.1 0-1.4-.6-1.5-.8a1 1 0 00-.7-.7.7.7 0 01-.6-.5v-.4h.3A1 1 0 105 12a1 1 0 01-.7-.1 1 1 0 01-.4-1v-.1a1.5 1.5 0 01.8-.9A1 1 0 005 8.5a1.3 1.3 0 01-.1-2 1 1 0 011-.2c.2 0 .4.2.6.7a1 1 0 101.7-.8c-.2-.5-.6-1-1-1.3a2 2 0 011.2-1l.6-.1v7.5l-.9-.2a1 1 0 100 1.8c.4 0 1 .7 1 1.7S8.4 16 8 16.2z" fill="#4F0599"/>
</svg>
```

**Review by dmzski — APPROVED** [review]

File: src/components/Icon/index.tsx [hunk 2]
```diff
@@ -8,6 +8,13 @@ import Box, { XStyledProps } from '../Box'
...
+      <path d="M10.838 14.581l-.01-9.844.002-.117a2.352 2.352 0 00-.992-2.406A2.753 2.753 0 007.744 2.2a4.042 4.042 0 00-2.511 2.277 2.75 2.75 0 00-1.74.927 3.038 3.038 0 00-.44 3.426 2.884 2.884 0 00-.998 1.809 2.547 2.547 0 00.786 2.418c.165.15.347.281.542.39-.107.477-.067.975.114 1.43a2.55 2.55 0 001.496 1.502A2.971 2.971 0 007.735 18c.14 0 .285-.009.435-.024a3.185 3.185 0 002.663-3.368c0-.01.005-.017.005-.026zM7.98 16.165c-1.095.105-1.378-.579-1.436-.779a.91.91 0 00-.766-.681.694.694 0 01-.584-.482.579.579 0 01-.017-.428c.104-.007.207-.02.31-.039a.909.909 0 00.227-1.675.911.911 0 00-.699-.083c-.21.037-.426.01-.62-.077a1.134 1.134 0 01-.11-.06l-.028-.02a.958.958 0 01-.407-.985c.005-.029.003-.047.01-.077v-.002a1.48 1.48 0 01.819-.84.91.91 0 00.335-1.472 1.277 1.277 0 01-.125-1.896 1.077 1.077 0 01.925-.31c.29.012.533.245.724.693a.91.91 0 101.675-.712 3.109 3.109 0 00-1.092-1.386 2.054 2.054 0 011.27-.953c.184-.068.378-.103.574-.102.052.257.066.52.044.781l-.003.14.007 6.592a2.205 2.205 0 00-.88-.199.91.91 0 100 1.82c.358 0 .88.645.88 1.653 0 .966-.52 1.525-1.033 1.579z" fill="#4F0599"/>
+    </g>
```
Comment (adriengibrat) [inline src/components/Icon/index.tsx:16]:
```suggestion
  ai: () => (
    <>
     <path d="M17.7 11.9v-.2l.1-.6a2.8 2.8 0 00-1-2.3 3 3 0 00-.4-3.4c-.5-.5-1.1-.8-1.8-1a4 4 0 00-2.5-2.2c-.7-.3-1.4-.3-2.1 0a2.3 2.3 0 00-1 2.4v10a3.2 3.2 0 002.7 3.4h.4a3 3 0 002.8-1.6 2.5 2.5 0 001.4-1.5c.2-.5.3-1 .2-1.5.5-.3 1-.9 1.2-1.5zm-3 2.3a.7.7 0 01-.6.5 1 1 0 00-.8.7 1.2 1.2 0 01-1.4.8c-.6 0-1-.6-1-1.6V12l.1-.7a.8.8 0 01.8-.5 1 1 0 000-1.8 3 3 0 00-1 .1V4.6v-.8l.7.1a2 2 0 011.2 1c-.5.3-.8.8-1 1.3a1 1 0 001.6.8c.2-.5.4-.7.7-.7a1 1 0 011 .3 1.3 1.3 0 01-.2 1.9 1 1 0 00.4 1.4l.8.8v.4l-.1.4-.2.2a1.1 1.1 0 01-.9.2 1 1 0 00-.5 1.8h.3a.6.6 0 010 .5z" fill="#AE5DF6"/>
    <path d="M10.8 14.6V4.7a2.4 2.4 0 00-1-2.5c-.6-.3-1.4-.3-2 0a4 4 0 00-2.6 2.3c-.6 0-1.3.4-1.7.9A3 3 0 003 8.8c-.6.5-1 1.1-1 1.8a2.5 2.5 0 00.7 2.5l.6.3c-.1.5 0 1 .1 1.5A2.5 2.5 0 005 16.4 3 3 0 007.7 18h.5a3.2 3.2 0 002.6-3.4zM8 16.2c-1.1 0-1.4-.6-1.5-.8a1 1 0 00-.7-.7.7.7 0 01-.6-.5v-.4h.3A1 1 0 105 12a1 1 0 01-.7-.1 1 1 0 01-.4-1v-.1a1.5 1.5 0 01.8-.9A1 1 0 005 8.5a1.3 1.3 0 01-.1-2 1 1 0 011-.2c.2 0 .4.2.6.7a1 1 0 101.7-.8c-.2-.5-.6-1-1-1.3a2 2 0 011.2-1l.6-.1v7.5l-.9-.2a1 1 0 100 1.8c.4 0 1 .7 1 1.7S8.4 16 8 16.2z" fill="#4F0599"/>
    </>
```

**Review by chambo-e — APPROVED** [review]

**Review by adriengibrat — APPROVED** [review]

**Pull request merged**